README.md

```diff
@@ -19,78 +26,71 @@ To install from source, follow the instructions in the [documentation](https://q
 
 ## Creating Your First Quantum Program in Qiskit Terra
 
-Now that Qiskit is installed, it's time to begin working with Terra.
+Now that Qiskit is installed, it's time to begin working with Qiskit. To do this
+we create a `QuantumCircuit` object to define a basic quantum program.
+
+```python
+from qiskit import QuantumCircuit
+qc = QuantumCircuit(2, 2)
+qc.h(0)
+qc.cx(0, 1)
+qc.measure([0,1], [0,1])
+```
 
-We are ready to try out a quantum circuit example, which is simulated locally using Qiskit's `BasicAer`. This is a simple example that makes an entangled state.
+This simple example makes an entangled state, also called a [Bell state](https://qiskit.org/textbook/ch-gates/multiple-qubits-entangled-states.html#3.2-Entangled-States-).
```

Comment:
```suggestion
This simple example makes an entangled state, also called [Bell state](https://qiskit.org/textbook/ch-gates/multiple-qubits-entangled-states.html#3.2-Entangled-States-).
```

Reply:
Are you sure this is correct? It reads a bit weird to me to have "an entangled state" followed by "bell state" without the article. But I'm hardly a grammar expert

Reply:
hmm... not sure actually. Let's keep yours.